server/meta.go

```diff
@@ -0,0 +1,129 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+
+	redis "gopkg.in/redis.v5"
+
+	"go.uber.org/zap"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/event"
+	"github.com/openfresh/plasma/log"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+func NewMetaServer(opt Option) *http.Server {
```

Comment:
__[golint]__
* [golint] exported function NewMetaServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->